packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -84,13 +84,23 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
             ) : (null) }
             { renderedItems.map(
               (item, index) => (
-                <li className='ms-Breadcrumb-listItem' key={ item.key || String(index) } ref={ item.key || String(index) } >
-                  <a className='ms-Breadcrumb-itemLink'
-                    onClick={ item.onClick ? this._onBreadcrumbClicked.bind(this, item) : null }
-                    href={ item.href }
-                    role={ item.onClick ? 'button' : 'link' }>
-                    { item.text }
-                  </a>
+                <li className='ms-Breadcrumb-listItem' key={ item.key || String(index) } ref={ item.key || String(index) }>
+                  { (() => {
+                    if (item.onClick) {
```

Comment:
is this correct? What if I don't have an `onClick` handler but I set an `href` prop because I want the browser to treat it like a plain link? We might guard on either `onClick` or `href`